### ISSUE TITLE
[HOLD for payment 2023-09-29] [$1000] Show different UI for a cancelled task

### ISSUE THREAD

**Issue by thienlnam** [Expensify/App#19504]
Right now, when a task is cancelled nothing about the task preview changes

<img width="716" alt="Screenshot 2023-05-23 at 4 18 48 PM" src="https://github.com/Expensify/App/assets/30609178/a173f0f8-df86-47c5-9ccf-7ef3edf69820">

Updates:

The task report is no longer archived. We will adjust the UI if a task parentReportID is deleted.
![image](https://github.com/Expensify/App/assets/30609178/15f47fae-8d38-4b78-8667-7686a25c12cf)

TODO: 
- Update the taskPreview to show [Deleted task]
- Update the copy from Cancel task to Delete Task
- Show the task detailed view in a 'thread' view
- Update the header to also show [Deleted task]

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01fbfabca6cddecf7f</li>
        <li>Upwork Job ID: 1665781150641840128</li>
        <li>Last Price Increase: 2023-06-05</li>
    </ul>
</details>

**Comment by shawnborton** [comment]
Can you start a Slack discussion for this? I think the strikethrough might not be a good idea because that's an offline pattern to indicate that something would be deleted

cc @JmillsExpensify @trjExpensify 

Also, I'm curious why a task can even be cancelled in the first place? Why not just delete it?

**Comment by thienlnam** [comment]
Started thread here: https://expensify.slack.com/archives/C04QEB4MJEQ/p1685984349292879

Seems like what we need to do here is:
Update the taskPreview to show [deleted task]

**Comment by sethraj14** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.

Show different UI for a cancelled task

### What is the root cause of that problem?

We use this to show cancel task - 
` canceled: 'Canceled task'`


### What changes do you think we should make in order to solve the problem?

Change the UI for cancelled task to this 

`canceled: '[Deleted task]',`

![Screenshot 2023-06-06 at 12 00 47 AM](https://github.com/Expensify/App/assets/14844259/056ce543-c6e1-477d-9368-373cbe150a5b)

### What alternative solutions did you explore? (Optional)

**Comment by JavSFDev** [comment]
Contributor details
Your Expensify account email: javsfdev323@gmail.com
Upwork Profile Link: https://www.upwork.com/freelancers/~01b7fc496875b2b3dd

**Comment by spiderman128** [comment]
Contributor details
Your Expensify account email: oldcrocodile128@gmail.com
Upwork Profile Link: https://www.upwork.com/freelancers/~0199bfb95677b75486

**Comment by spiderman128** [comment]
Contributor details
Your Expensify account email: talent111dev@gmail.com
Upwork Profile Link: https://www.upwork.com/freelancers/~0199bfb95677b75486

**Comment by greg-schroeder** [comment]
Hey @thienlnam - just to make sure I'm clear. This is still an `External` issue, but only once you've made changes that are essentially `Internal`?

**Comment by thienlnam** [comment]
Yeah that's correct

**Comment by greg-schroeder** [comment]
Okay, got it. Pending Jack's next move then!

**Comment by greg-schroeder** [comment]
This is still held on Jack's changes

**Comment by sophiepintoraetz** [comment]
Hmm @greg-schroeder @thienlnam - I have this UI inconsistency here https://github.com/Expensify/App/issues/21674 where the space changes when you cancel a task and navigate away from the chat and back to it - will the impending changes fix this too?
![image](https://github.com/Expensify/App/assets/42940078/cfde346c-be11-4e3f-a76e-2120a185739e)

**Comment by thienlnam** [comment]
Yup, this will also solve that since 'canceling a task' won't actually archive the room. Feel free to put a hold on all bugs related to the Cancel Task flow

**Comment by greg-schroeder** [comment]
Thanks for that insight @thienlnam

**Comment by greg-schroeder** [comment]
This remains on hold

**Comment by greg-schroeder** [comment]
Holding still as this is part of the cancel task flow

**Comment by greg-schroeder** [comment]
""

**Comment by thienlnam** [comment]
Been meaning to get around to this - it's kind of a big change in the BE, but currently focused on https://github.com/Expensify/Expensify/issues/294203.

**Comment by greg-schroeder** [comment]
Thanks for the context @thienlnam

**Comment by greg-schroeder** [comment]
Draft PR is up

**Comment by greg-schroeder** [comment]
Hey @thienlnam is this issue complete? It seems the PR was deployed to Prod... and also seems like maybe it was 100% Internal?

**Comment by thienlnam** [comment]
The issue requires BE and FE changes - I'm currently working on the App (FE) changes in https://github.com/Expensify/App/pull/24137

Mostly internal, but we'll still need to pay C+ for reviewing the App PR when it is ready

**Comment by greg-schroeder** [comment]
Gotcha 👍

**Comment by greg-schroeder** [comment]
Draft PR continues to be worked on

**Comment by thienlnam** [comment]
Heads down in wave 5 this week so haven't been able to get around to this

**Comment by greg-schroeder** [comment]
Sounds like Jack will get to this soon!

**Comment by greg-schroeder** [comment]
Sorry - can you help me understand where we're at on this one @thienlnam?

**Comment by thienlnam** [comment]
App PR in review!

**Comment by greg-schroeder** [comment]
This is internal and there's no issue reporter. So only @mollfpr requires payment. Looks like that'll be $1,000 - you can make a manual request!

**Comment by greg-schroeder** [comment]
I don't think we need the traditional C+ checklist for this, but please let me know if you disagree. Closing this out as payment will be handled via NewDot.

**Comment by JmillsExpensify** [comment]
$1,000 payment approved for @mollfpr based on comment above.
